src/www/ui/showjobs.php

```diff
@@ -147,11 +141,10 @@ public function Output()
 
     $this->vars['uploadId']= $uploadPk;
     // micro menus
-    $v .= menu_to_1html(menu_find($this->Name, $MenuDepth),0);
+    menu_to_1html(menu_find($this->Name, $MenuDepth),0);
```

Comment:
The variable `$v` has been unused since the commit 674710e5d92556b86ace9bfc572c98b15caf988e